lib/devcycle-ruby-server-sdk/localbucketing/config_manager.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require 'sorbet-runtime'
+require 'concurrent-ruby'
+require 'typhoeus'
+require 'json'
+
+module DevCycle
+  class ConfigManager
+    extend T::Sig
+
+    sig { params(sdkKey: String, local_bucketing: LocalBucketing).returns(NilClass) }
+    def initialize(sdkKey, local_bucketing)
+      @first_load = true
+      @local_bucketing = local_bucketing
+      @sdkKey = sdkKey
+      @config_e_tag = ""
+      @config_poller = Concurrent::TimerTask.new(
+        { execution_interval: @local_bucketing.options.config_polling_interval_ms / 1000, run_now: true }) do |task|
+        fetch_config(false, task)
```

Comment:
I wonder if there is a better way to format this? 

```suggestion
      @config_poller = Concurrent::TimerTask.new(
        { execution_interval: @local_bucketing.options.config_polling_interval_ms / 1000, run_now: true }
      ) do |task|
        fetch_config(false, task)
```

Reply:
so does `run_now` basically execute the first run synchronously here?

Reply:
No, still async, just forces it. I've gotta put in the sync call too

Reply:
ok yea we need to call `fetch_config` synchronously here as part of this function.

Reply:
I guess the question is do we want to force a full sync request of the config here or not. In Node it triggers the fetch call but doesn't wait for the response because you can't have a async constructor function: https://github.com/DevCycleHQ/js-sdks/blob/main/sdk/nodejs/src/environmentConfigManager.ts#L43

Reply:
It's a good question, I think this would be a good test harness setup?

Ultimately it should be sending a config request as soon as possible, to prevent a delay in evaluating variables.